map.py

```diff
@@ -0,0 +1,78 @@
+import dash
+import dash_html_components as html
+import dash_leaflet as dl
+import dash_leaflet.express as dlx
+import pandas as pd
+import json
+
+from dash.dependencies import Output, Input
+
+keys = ["watercolor", "toner", "terrain"]
+url_template = "http://{{s}}.tile.stamen.com/{}/{{z}}/{{x}}/{{y}}.png"
+attribution = 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, ' \
+              '<a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data ' \
+              '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+
+df = pd.read_csv("data/elections/2020-general-election.csv")
+
+with open("data/geojson/stl-city/wards.json") as read_file:
+	ward_geojson = json.load(read_file)
+	for ward_object in ward_geojson["features"]:
+		df_ward = df[df["Ward"] == ward_object["properties"]["Ward"]]
+		ward_object["properties"]["Turnout"] = df_ward.iloc[0]["Turnout"]
```

Comment:
I got a little confused in dealing with Python's data structures, so I'm adding the turnout to the geojson here. It's probably better to do this in the `get_info` though, since we might be varying what field we're grabbing from the CSV